app/src/main/res/layout/quest_location.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingStart="24dp"
+    android:paddingEnd="24dp">
+<TextView
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:text="@string/quest_defibrillator_location_description"
+    android:labelFor="@+id/commentInput"/>
+
+<EditText xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/locationInput"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:inputType="text"
+    android:maxLength="253"
+    />
```

Comment:
Wouldn't it make sense to make it a multiline edit text?

Reply:
Sure, multiline looks better, i changed that now.

![Screenshot_20231021-214557_1](https://github.com/streetcomplete/StreetComplete/assets/44033795/432c51ed-3192-41df-8775-df4c6a452c16)

Reply:
I notice that you put "in the porter's lounge" but these do not show up in the string in the app.

To fix that, I think what should work is to put the entire string in "..." and escape the \"in the porter's lounge\"

Reply:
Oops, i didn't notice the missing quotation marks. 

> To fix that, I think what should work is to put the entire string in "..." and escape the \"in the porter's lounge\"

Thank you, this worked fine.
![Screenshot_20231022-000318(1)](https://github.com/streetcomplete/StreetComplete/assets/44033795/ffb15c95-e5cf-4510-82a3-d4baa29faa4b)